src/js/gameBuilder.js

```diff
@@ -4,38 +4,34 @@ import {
     onTouchStart,
     onTouchEnd
  } from "./actions.js";
+import { removeGameTable } from "./gameState";
 
-export function buildGameTable(level) {
-    let rows = level.rows;
-    let cols = level.cols;
-    let maxMines = level.maxMines;
-    let tableCells = {};
-    let table = {
+const fragment = document.createDocumentFragment();
+const gameTableElement = document.getElementById("gameTable");
+
+gameTableElement.addEventListener("click", clickOnCell);
```

Comment:
We install the event listener(s) only once and don't need to bother any more in the future. This way everything renders and builds faster as installing event listeners is heavy (9 * 9 * 4 = 324 listeners for the easy game, or ... only 4 for all games)